commit 9db8cd14ffcd03d0268f669da11c9eeef654bae7 (refs/changes/62/5162/1)
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2018-11-29T11:27:07-05:00 (10 months ago)
    
    TOOLS-2117 sdcnode updates for November 2018 node.js security releases
    TOOLS-2068 sdcnode updates for August 2018 node.js security releases

diff --git a/nodeconfigs.json b/nodeconfigs.json
index 517eff7..1e4d04c 100644
--- a/nodeconfigs.json
+++ b/nodeconfigs.json
@@ -1,4 +1,39 @@
 [
+    {
+        "// image": "minimal-multiarch 18.1.0",
+        "image": "1ad363ec-3b83-11e8-8521-2f68a4a34d5d",
+        "version": "v8.14.0",
+        "patch_files": [
+            "patches/tls-reload-certs-v8.x.patch"
+        ],
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
+        "gcc_version_prefix": "4.9.",
+        "rpath": "$ORIGIN/../lib",
+        "include_gcc_runtime_libs": true,
+        "build_tag": "gz"
+    },
+    {
+        "// image": "minimal-multiarch 18.1.0",
+        "image": "1ad363ec-3b83-11e8-8521-2f68a4a34d5d",
+        "comment": "For use by services running in zones. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
+        "version": "v8.14.0",
+        "patch_files": [
+            "patches/tls-reload-certs-v8.x.patch"
+        ],
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
+        "build_tag": "zone"
+    },
+    {
+        "// image": "minimal-multiarch 18.1.0",
+        "image": "1ad363ec-3b83-11e8-8521-2f68a4a34d5d",
+        "comment": "For use by services running in zones. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
+        "version": "v8.14.0",
+        "patch_files": [
+            "patches/tls-reload-certs-v8.x.patch"
+        ],
+        "configure_flags": "--without-snapshot --dest-cpu=x64 --with-dtrace",
+        "build_tag": "zone64"
+    },
     {
         "// image": "minimal-multiarch 18.1.0",
         "image": "1ad363ec-3b83-11e8-8521-2f68a4a34d5d",
@@ -139,6 +174,41 @@
         "configure_flags": "--without-snapshot --dest-cpu=x64 --with-dtrace",
         "build_tag": "zone64"
     },
+    {
+        "// image": "minimal-multiarch 18.1.0",
+        "image": "1ad363ec-3b83-11e8-8521-2f68a4a34d5d",
+        "version": "v6.15.0",
+        "patch_files": [
+            "patches/tls-reload-certs-v6.x.patch"
+        ],
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
+        "gcc_version_prefix": "4.9.",
+        "rpath": "$ORIGIN/../lib",
+        "include_gcc_runtime_libs": true,
+        "build_tag": "gz"
+    },
+    {
+        "// image": "minimal-multiarch 18.1.0",
+        "image": "1ad363ec-3b83-11e8-8521-2f68a4a34d5d",
+        "comment": "For use by services running in zones. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
+        "version": "v6.15.0",
+        "patch_files": [
+            "patches/tls-reload-certs-v6.x.patch"
+        ],
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
+        "build_tag": "zone"
+    },
+    {
+        "// image": "minimal-multiarch 18.1.0",
+        "image": "1ad363ec-3b83-11e8-8521-2f68a4a34d5d",
+        "comment": "For use by services running in zones. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
+        "version": "v6.15.0",
+        "patch_files": [
+            "patches/tls-reload-certs-v6.x.patch"
+        ],
+        "configure_flags": "--without-snapshot --dest-cpu=x64 --with-dtrace",
+        "build_tag": "zone64"
+    },
     {
         "// image": "minimal-multiarch 18.1.0",
         "image": "1ad363ec-3b83-11e8-8521-2f68a4a34d5d",
@@ -209,6 +279,41 @@
         "configure_flags": "--without-snapshot --dest-cpu=x64 --with-dtrace",
         "build_tag": "zone64"
     },
+    {
+        "// image": "sdc-minimal-multiarch-lts 15.4.1",
+        "image": "18b094b0-eb01-11e5-80c1-175dac7ddf02",
+        "version": "v6.15.0",
+        "patch_files": [
+            "patches/tls-reload-certs-v6.x.patch"
+        ],
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
+        "gcc_version_prefix": "4.9.",
+        "rpath": "$ORIGIN/../lib",
+        "include_gcc_runtime_libs": true,
+        "build_tag": "gz"
+    },
+    {
+        "// image": "sdc-minimal-multiarch-lts 15.4.1",
+        "image": "18b094b0-eb01-11e5-80c1-175dac7ddf02",
+        "comment": "For use by services running in zones. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
+        "version": "v6.15.0",
+        "patch_files": [
+            "patches/tls-reload-certs-v6.x.patch"
+        ],
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
+        "build_tag": "zone"
+    },
+    {
+        "// image": "sdc-minimal-multiarch-lts 15.4.1",
+        "image": "18b094b0-eb01-11e5-80c1-175dac7ddf02",
+        "comment": "For use by services running in zones. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
+        "version": "v6.15.0",
+        "patch_files": [
+            "patches/tls-reload-certs-v6.x.patch"
+        ],
+        "configure_flags": "--without-snapshot --dest-cpu=x64 --with-dtrace",
+        "build_tag": "zone64"
+    },
     {
         "// image": "sdc-minimal-multiarch-lts 15.4.1",
         "image": "18b094b0-eb01-11e5-80c1-175dac7ddf02",
